easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'FLASH2'
+version = '20160507'
+versionsuffix = '64826f4' # git commit to download
+
+homepage = 'https://github.com/dstreett/FLASH2'
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software tool 
+ to merge paired-end reads from next-generation sequencing experiments.
+
+ Flash2 has some improvements from flash_1 including new logic from innie and outie overlaps as well 
+ as some initial steps for flash for amplicons
+"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/dstreett/FLASH2/archive/']
+sources = ['%(versionsuffix)s.tar.gz']
+
+dependencies = [
+    ('zlib', '1.2.8'),
+]
+
+files_to_copy = [(['flash2'], 'bin'), 'README']
+                
+sanity_check_paths = {
+    'files': ['bin/flash2'],
+    'dirs': [],
+}
+
+modloadmsg = "This app uses all available cpus in the machine by default!! PLEASE BE SURE TO USE THE --threads FLAG!"
```

Comment:
what does `--threads` do? why not mention that (it's OK if not everyone 100% understands imho)